packages/headless/src/features/facets/category-facet-set/interfaces/request.ts

```diff
@@ -26,6 +26,10 @@ export interface CategoryFacetRequest
     Delimitable,
     Type<'hierarchical'>,
     SortCriteria<CategoryFacetSortCriterion> {
+  /** @default 5 */
+  numberOfValues: number;
+  /** @default ; */
+  delimitingCharacter: string;
```

Comment:
This is a really cool feature of jsdoc. In order to override the default character, one only needs to rewrite the option, and add the `@default` tag. This will change the default, while still displaying the description on the base interface.